spec/views/spree/admin/orders/index.html.haml_spec.rb

```diff
@@ -10,6 +10,12 @@
   end
 
   before do
+    controller.singleton_class.class_eval do
+      def current_ability
+        Spree::Ability.new(Spree.user_class.new)
+      end
+    end
```

Comment:
this is the solution to the Ability problem described earlier ☝️

Reply:
Interesting. What is `controller` here? Is that just a basic test controller that doesn't know anything about our app?

That would make sense to me. Then the problem would be that a helper is relying on a controller method that isn't there until you add it here? That sounds like a design flaw causing a bad dependency.

Reply:
yes, I think this controller is just a test controller that can be changed like this.
I got this from here:
https://relishapp.com/rspec/rspec-rails/v/2-0/docs/view-specs/view-spec#spec-with-view-that-accesses-helper-method-helpers

This is testing the view and we can manipulate/instrument the "mock" controller like this.